tests/mocks/types_handler.go

```diff
@@ -5,44 +5,43 @@
 package mocks
 
 import (
+	reflect "reflect"
+
 	types "github.com/cosmos/cosmos-sdk/types"
 	gomock "github.com/golang/mock/gomock"
-	reflect "reflect"
 )
 
-// MockAnteDecorator is a mock of AnteDecorator interface
+// MockAnteDecorator is a mock of AnteDecorator interface.
 type MockAnteDecorator struct {
 	ctrl     *gomock.Controller
 	recorder *MockAnteDecoratorMockRecorder
 }
 
-// MockAnteDecoratorMockRecorder is the mock recorder for MockAnteDecorator
+// MockAnteDecoratorMockRecorder is the mock recorder for MockAnteDecorator.
 type MockAnteDecoratorMockRecorder struct {
 	mock *MockAnteDecorator
 }
 
-// NewMockAnteDecorator creates a new mock instance
+// NewMockAnteDecorator creates a new mock instance.
 func NewMockAnteDecorator(ctrl *gomock.Controller) *MockAnteDecorator {
 	mock := &MockAnteDecorator{ctrl: ctrl}
 	mock.recorder = &MockAnteDecoratorMockRecorder{mock}
 	return mock
 }
 
-// EXPECT returns an object that allows the caller to indicate expected use
+// EXPECT returns an object that allows the caller to indicate expected use.
 func (m *MockAnteDecorator) EXPECT() *MockAnteDecoratorMockRecorder {
 	return m.recorder
 }
 
-// AnteHandle mocks base method
+// AnteHandle mocks base method.
 func (m *MockAnteDecorator) AnteHandle(ctx types.Context, tx types.Tx, simulate bool, next types.AnteHandler) (types.Context, error) {
 	m.ctrl.T.Helper()
-	ret := m.ctrl.Call(m, "AnteHandle", ctx, tx, simulate, next)
-	ret0, _ := ret[0].(types.Context)
-	ret1, _ := ret[1].(error)
-	return ret0, ret1
+	m.ctrl.Call(m, "AnteHandle", ctx, tx, simulate, next)
+	return next(ctx, tx, simulate)
```

Comment:
The `next` is not called in a generated code.  So we have to customize the code. Since the test was not functioning, this function never worked as expected.

SOLUTION: I'm removing `types_handlers` from auto - generated code.

Reply:
@fdymylja  - you were looking at the mocgen - did you solve it in any other way?